modules/simple_forms_api/app/services/simple_forms_api/pdf_filler.rb

```diff
@@ -9,6 +9,8 @@ class PdfFiller
     TEMPLATE_BASE = Rails.root.join('modules', 'simple_forms_api', 'templates')
 
     def initialize(form_number:, form:, name: nil)
+      raise "form_number and form are required" if form_number.blank? || !form&.data
```

Comment:
Nice! What do you think of splitting this into two lines:
```ruby
raise 'form_number is required' if form_number.blank?
raise 'form needs a data attribute' unless form&.data
```
That way if we go error spelunking later we'll have a more precise error message.

Reply:
I like that idea, fixed! 👍